build/teamcity/internal/cockroach/nightlies/private_roachtest_impl.sh

```diff
@@ -10,6 +10,11 @@ set -exuo pipefail
 
 dir="$(dirname $(dirname $(dirname $(dirname $(dirname "${0}")))))"
 
+# N.B. export variables like `root` s.t. they can be used by scripts called below.
+set -a
+source "$dir/teamcity-support.sh"
+set +a
+
 source "$dir/teamcity-support.sh"
```

Comment:
Is this line still needed?

Reply:
Nope 🤦 It was really missing `set -a` to export `root`.